resources/user-agents/crawlers/general-crawlers/general-crawlers.json

```diff
@@ -2240,6 +2240,102 @@
         {
           "match": "AdsTxtCrawler/*",
           "browser": "ads-txt-crawler"
+        },
+        {
+          "match": "Mozilla/5.0 (compatible; Cincraw/*",
+          "browser": "cincraw"
+        },
+        {
+          "match": "lotayabot",
+          "browser": "lotayabot"
+        },
+        {
+          "match": "NativeAIBot",
+          "browser": "native-ai-bot"
+        },
+        {
+          "match": "SirdataBot",
+          "browser": "sirdata-bot"
+        },
+        {
+          "match": "beBee bot (support@bebee.com)",
+          "browser": "be-bee-bot"
+        },
+        {
+          "match": "Mozilla/5.0 (compatible; Onespot-ScraperBot/*",
+          "browser": "onespot"
+        },
+        {
+          "match": "Mozilla/5.0 (compatible; Seekport Crawler; http://seekport.com/*",
+          "browser": "seekport-crawler"
+        },
+        {
+          "match":  "BrightEdge Crawler/*",
+          "browser":  "bright-edge-crawler"
+        },
+        {
+          "match":  "BublupBot (+https://www.bublup.com/bublup-bot.html)",
+          "browser":  "bublup-bot"
+        },
+        {
+          "match":  "ISS_CRAWLER_V*",
+          "browser":  "iss-crawler"
+        },
+        {
+          "match":  "KareBot/*",
+          "browser":  "kare-bot"
+        },
+        {
+          "match":  "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.102 Safari/537.36 MicroAdBot/*",
```

Comment:
Please look into the Chrome files to make this rule less specific for a version.